src/Core/Payment/PaymentModulePreferencesConfiguration.php

```diff
@@ -35,6 +35,9 @@
  */
 final class PaymentModulePreferencesConfiguration implements DataConfigurationInterface
 {
+    public const CUSTOMER_CURRENCY = -1;
```

Comment:
It's a good thing to introduce these consts, but they are only used in the form for no. SO they have no relation with the actual behaviour. I guess in the legacy code those values are used hard-coded Ideally you should replace all the hard-coded values with these consts.